packages/ui/react-ui-deck/src/components/Deck/Deck.tsx

```diff
@@ -123,25 +123,21 @@ const DeckPlankRoot = ({
   );
 };
 
-type DeckPlankProps = ThemedClassName<ComponentPropsWithRef<'article'>> & {
-  scrollIntoViewOnMount?: boolean;
-  suppressAutofocus?: boolean;
-};
+type DeckPlankProps = ThemedClassName<ComponentPropsWithRef<'article'>>;
 
 type DeckPlankResizing = Pick<MouseEvent, 'pageX'> & { size: number } & { [Unit in DeckPlankUnit]: number };
 
 const DeckPlankContent = forwardRef<HTMLDivElement, DeckPlankProps>(
-  ({ classNames, style, children, scrollIntoViewOnMount, suppressAutofocus, ...props }, forwardedRef) => {
+  ({ classNames, style, children, ...props }, forwardedRef) => {
     const [isSm] = useMediaQuery('sm', { ssr: false });
     const articleElement = useRef<HTMLDivElement | null>(null);
     const ref = useComposedRefs(articleElement, forwardedRef);
     const { unit = 'rem', size = PLANK_DEFAULTS.size } = usePlankContext('DeckPlankContent');
     const { solo } = useDeckContext('DeckPlankContext');
     const inlineSize = solo ? undefined : isSm ? `${size}${unit}` : '100dvw';
 
-    // TODO(burdon): Experiment to fade in. Works well, except when toggling between solo mode.
     // Opacity transition prevents flicker when content is first rendered.
-    // const [visible, setVisible] = useState('opacity-0 transition duration-1000');
+    const [visible, setVisible] = useState<string>();
```

Comment:
`visible` is a different property that isn’t transitionable, consider `opaque` or `transparent`.

Reply:
i am using opacity!?